Gemfile.jruby-2.5.lock.release

```diff
@@ -882,15 +882,9 @@ DEPENDENCIES
   logstash-integration-jdbc
   logstash-integration-kafka
   logstash-integration-rabbitmq
-  logstash-mixin-aws
```

Comment:
These are transitive dependencies from various plugins, why was removed?

Reply:
The DEPENDENCIES section of the lock file lists dependencies contained in the Gemfile. Dependencies like mixin-aws are still included:

```
~/elastic/logstash bump_8_3_0*
❯ git co update_lock_1648639558
Branch 'update_lock_1648639558' set up to track remote branch 'update_lock_1648639558' from 'origin'.
Switched to a new branch 'update_lock_1648639558'

~/elastic/logstash update_lock_1648639558*
❯ ./gradlew clean installDefaultGems

[...]

~/elastic/logstash update_lock_1648639558*
❯ find . -name "logstash-mixin-aws*"
./vendor/bundle/jruby/2.5.0/specifications/logstash-mixin-aws-4.4.1.gemspec
./vendor/bundle/jruby/2.5.0/cache/logstash-mixin-aws-4.4.1.gem
./vendor/bundle/jruby/2.5.0/gems/logstash-mixin-aws-4.4.1
./vendor/bundle/jruby/2.5.0/gems/logstash-mixin-aws-4.4.1/logstash-mixin-aws.gemspec

~/elastic/logstash update_lock_1648639558*
❯ grep logstash-mixin-aws Gemfile.lock 
      logstash-mixin-aws (>= 4.3.0)
      logstash-mixin-aws (>= 4.3.0)
    logstash-mixin-aws (4.4.1)
      logstash-mixin-aws (>= 1.0.0)
      logstash-mixin-aws (>= 4.3.0)
      logstash-mixin-aws (>= 1.0.0)
      logstash-mixin-aws (>= 4.3.0)

```

Because the previous bump process "manually" edited the file, it left these entries in place for years. but now that we're letting bundler control the update it cleans up these entries that aren't needed.